faq.md

```diff
@@ -27,13 +26,12 @@ gantt
     Publication in the Official Journal of the EU (November 20, 2024): milestone, 2024-11-20, 5m
     Entry into force (December 11, 2024): milestone, 2024-12-11, 5m
     Implementation phase: 2024-12-11, 3y
-    Reporting obligations of manufacturers (September 11, 2026): milestone, 2026-09-11, 5m
+    Reporting obligations of vulnerabilities and incidents (September 11, 2026): milestone, 2026-09-11, 5m
```

Comment:
exploited vulnerabilities

This is an important difference

Reply:
Oh wow. Is that really the actual phrasing? The word "exploited" there changes everything.

Reply:
Yes, as used above in FAQ itself. This is a shorthand for the diagram.